### DISCUSSION TITLE
How do I fix this?

### DISCUSSION THREAD

**Discussion by wktra** [Nevysha/Cozy-Nest#162] [General]
How do i turn off themes? 
![Screenshot 2023-06-30 172128](https://github.com/Nevysha/Cozy-Nest/assets/117292904/a26cc91d-cf09-407a-903f-caadcf6c4499)

**Comment by Nevysha** [comment]
remove __theme from url
and use Cozy Nest settings to switch from light/dark
settings are here in top right corner : 
![image](https://github.com/Nevysha/Cozy-Nest/assets/122687716/45020b7f-b6ad-4945-91ba-7aea3762f4df)
it should look like "127.0.0.1:7860?__theme=light" or something
just remove the __theme=light if you use Cozy Nest

**Reply by Chubbly** [reply]
Extra info for folks:

You need to have your theme set as gradio/default (instead of black orange if using Vladmatic/SD.Next), otherwise it will always append __theme to the URL on load and mess up Cozy.
